### ISSUE TITLE
Button gets triggered accidentally when swiping for scrolling on mobile

### ISSUE THREAD

**Issue by Feuerhamster** [Nerwyn/service-call-tile-feature#16]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:** Current

**Last working release (if known):** /

**Browser and Operating System:**
Android 13 with HomeAssistant App

**Description of problem:**

Tile card feature buttons are getting triggered by accident when swiping to scroll.
I posted a video showing the issue: https://youtu.be/fRQNLYEdWv4.
The button should only be triggered if actually pressed. I guess there is something wrong with the event listeners on the button.

**Javascript errors shown in the web inspector (if applicable):** /

**YAML of tile card:**

```yaml
features:
  - type: custom:service-call
    entries:
      - type: button
        icon: mdi:power
        tap_action:
          action: call-service
          service: switch.toggle
          target:
            entity_id: switch.fernsehbank_steckdosenleiste_schalter
type: tile
entity: switch.fernsehbank_steckdosenleiste_schalter
name: TV Bank
icon: mdi:power-socket-eu
state_content: state
vertical: false
show_entity_picture: false

```

**Comment by Nerwyn** [comment]
That's a weird one, nice find. I'm able to recreate it on my phone and on desktop browsers when using the device toolbar in developer tools.

I believe it's an issue with touch events triggering when swiping to scroll, something I hadn't accounted for. I think I can use `touchmove` to detect scrolling and not fire actions if there is scrolling.

**Comment by Nerwyn** [comment]
Should be fixed in [3.1.1](https://github.com/Nerwyn/service-call-tile-feature/releases/tag/3.1.1). Fixing it for buttons/selectors was pretty easy, but sliders proved difficult. If your finger moves when you're tapping on a button or selector option the feature will now consider it a swipe and not call an action. This doesn't work for slider though, since you're supposed to be able to swipe on them. My solution for sliders is as long as the x distance is less than the y distance while swiping it'll consider it a slider action and not a scroll swipe.

**Comment by Feuerhamster** [comment]
Thank you very much 👍
